.circleci/config.yml

```diff
@@ -96,9 +96,11 @@ commands:
         default: false
     steps:
     - run: |
-       cargo build <<#parameters.release>>--release<</parameters.release>> --no-default-features --features="${FEATURES}" -p <<parameters.package>> ||
-       cargo build <<#parameters.release>>--release<</parameters.release>> --no-default-features --features="${FEATURES}" -p <<parameters.package>>
-
+            cmd="cargo build <<#parameters.release>>--release<</parameters.release>> --no-default-features --features="${FEATURES}" -p <<parameters.package>>"
+            for i in {1..16}; do
+                $cmd && exit 0
```

Comment:
that seems excessive

Reply:
Yeah, but don't worry, this was an old version, now we only repeat it a few times:

```bash
for i in 36 4 2 1 1; do
   ...
```

where `i` is the number of threads of the command.

It is to avoid job failures because of OOM errors.

Reply:
I think you're reviewing an older commit